### DISCUSSION TITLE
[Feature]: Does CloudNativePG have horizontalPodAutoScaler feature?

### DISCUSSION THREAD

**Discussion by ember11498** [cloudnative-pg/cloudnative-pg#5968] [Q&A]
### Is there an existing issue already for this feature request/idea?

- [x] I have searched for an existing issue, and could not find anything. I believe this is a new feature request to be evaluated.

### What problem is this feature going to solve? Why should it be added?

This might be a bad question as I am new to cnpg, but does cnpg have horizontalPodAutoScaler ? like we could specify a range of replicas and it would dinamically change accordingly to the requests load?.

So i would say like min 3 max 10 replicas and cnpg would create pods of pg replicas accordingly

### Describe the solution you'd like

add horizontalautoscaler

### Describe alternatives you've considered

none

### Backport?

Yes

### Are you willing to actively contribute to this feature?

No

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by gbartolini** [comment]
Just to understand: what is your goal here? Why would you need 10 replicas?

**Comment by ember11498** [comment]
@gbartolini more replicas means more read capacity no? if i have 10 replicas each have max connections of 200 then i can have 2000 concurrent connections. if i only have 2 replicas i have 400 max connections instead of 2000
